src/netcdf_writer_coordinates.jl

```diff
@@ -122,7 +122,7 @@ function target_coordinates(
     Union{Spaces.CenterFiniteDifferenceSpace, Spaces.FaceFiniteDifferenceSpace},
 }
     if disable_vertical_interpolation
-        return Array(parent(Fields.coordinate_field(space).z))[:, 1]
+        return Array(parent(space.grid.center_local_geometry.coordinates))[:, 1]
```

Comment:
```suggestion
        cspace = Spaces.space(space, Grids.CellCenter())
        return Array(parent(Fields.coordinate_field(cspace).z))[:, 1]
```
I think this should work?

Reply:
I think it does, but I will check

Reply:
Green (but perf): https://buildkite.com/clima/climaatmos-ci/builds/18366